apps/consumer-client/pages/zuzalu-examples/group-proof.tsx

```diff
@@ -70,33 +55,13 @@ export default function Page() {
               PASSPORT_URL,
               window.location.origin + "/popup",
               SEMAPHORE_GROUP_URL,
-              "1337",
-              "12345",
-              serverProving
+              "consumer-client"
```

Comment:
i think for completeness you should check that the PCD you receive matches this website name - `consumer-client`